tests/functional_tests/test_providers.py

```diff
@@ -45,6 +45,21 @@
     PROVIDER_NAME_WIDTH = 0
 
 
+class FastAPIServer(liveandletdie.Base):
+    def __init__(self, *args, **kwargs):
+        kwargs['executable'] = 'uvicorn'
+        super().__init__(*args, **kwargs)
+
+    def create_command(self):
+        return [
+            self.executable,
+            '{}:app'.format(self.path),
+            '--host {}'.format(self.host),
+            '--port {}'.format(self.port),
+            '--reload',
+        ]
+
+
```

Comment:
This should probably be a separate commit to the liveandletdie project (which is part of the authomatic org, so we can accept the PR easily enough).

Reply:
@mrichar1 Yes, it should, however, I saw that it's 2 years old project and no longer maintained and this is why I added it here, however, we can directly import it from liveandletdie and I'll handle the PR there atm, we can skip this one. Let me PR there real quick

Reply:
That's a new `liveandletdie` released.

Reply:
@mrichar1 Great, now I pushed here and fixed the issues. Have a look.